bjoern/request.c

```diff
@@ -205,7 +205,18 @@ on_message_complete(http_parser* parser)
   /* SERVER_NAME and SERVER_PORT */
   if (REQUEST->server_info->host) {
     _set_header(_SERVER_NAME, REQUEST->server_info->host);
-    _set_header(_SERVER_PORT, REQUEST->server_info->port);
+    if (REQUEST->server_info->port == Py_None) {
+      _set_header(_SERVER_PORT,
```

Comment:
I guess these environment items could be moved to `_initialize_request_module` as they never change? Also, please remove these line breaks.

Reply:
They could  be set in theory. But `_initialize_request_module` does not have "access" to the real values as the host name or port number can differ in different setups.

 As for the line breaks: I guess you mean I should write
```c
_set_header(_SERVER_PORT,  PyUnicode_FromFormat("%i", REQUEST->server_info->port));
```
instead of
```c
_set_header(_SERVER_PORT,
            PyUnicode_FromFormat("%i", REQUEST->server_info->port));
```
shouldn't I?

Reply:
> But _initialize_request_module does not have "access" to the real values as the host name or port number can differ in different setups.

You can simply pass `&info` to that function (see `_bjoernmodule.c`). They won't change while the server is running, or am I missing something?

> As for the line breaks

Yes that's what I meant. 90-100 chars line width is fine.

Reply:
You your are right, that would be possible.